pulp_2to3_migration/app/pre_migration.py

```diff
@@ -289,19 +271,33 @@
                   'type': repo_id_to_type[record.repo_id],
                   'is_migrated': False})
 
+    if not created:
+        # if it was marked as such because it was not present in the migration plan
+        repo.not_in_pulp2 = False
```

Comment:
Maybe a better name is needed.
Currently this flag is used to signify that a repo is either not in pulp 2 or not a part of the most recent migration plan.

Reply:
what about - not_in_current_migration_plan or in_current_migration, something alone the lines

Reply:
what about not_in_plan?

Reply:
not_in_plan seems the most descriptive

Reply:
+1 not_in_plan

Reply:
renamed

Reply:
renamed